src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, useContext } from 'react';
+import { AppTodoContext } from '../AppTodoContext/AppTodoContext';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export const TodoList: FC = () => {
+  const {
+    visibleTodos,
+    tempTodo,
+  } = useContext(AppTodoContext);
+
+  return (
+    <section className="todoapp__main">
+      {visibleTodos.map(todo => (
+        <TodoItem
+          key={todo.id}
+          todo={todo}
+        />
+      ))}
+
+      {tempTodo && (
+        <TodoItem
+          todo={tempTodo}
```

Comment:
Consider renaming the 'isItTempTodo' prop to 'isTempTodo' for better readability and consistency 📖